packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/ExecuteRulesTests.java

```diff
@@ -304,6 +305,7 @@ public void executeRules_privateFolderAsCollaborator()
     /**
      * Try to execute rule with broken action and receive 500.
      */
+    @Ignore("It shouldn't be possible to create a rule with broken action any more.")
```

Comment:
It's bound to be possible still. E.g. We can do loads of stuff by executing a script.  I can't think of a simple way to do this though, so maybe delete this test and create another ticket for us to work on in parallel to the other validation work.

Reply:
Will do, thx.

Reply:
Added ACS-3699